jcr_root/etc/designs/community/sitethemes/bootstrap-custom/clientlibs/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:mixinTypes="[cq:ReplicationStatus]"
     jcr:primaryType="cq:ClientLibraryFolder"
     categories="[cq.social.console.theme.bootstrap-custom]"
-    dependencies="[cq.social.console.theme.scfskin]"/>
+    dependencies="[cq.social.bootstrap.3,cq.social.console.theme.scfskin]"/>
```

Comment:
This seems to be required for a lot of the downloaded themes. The css doesn't include the bootstrap it just gets loaded after it.